contracts/strategy/liquity/LiquityStrategy.sol

```diff
@@ -151,18 +151,18 @@ contract LiquityStrategy is
         stabilityPool = IStabilityPool(_stabilityPool);
         curveExchange = ICurveExchange(_curveExchange);
         lqty = IERC20(_lqty);
-        minPrincipalProtectionPct = _principalProtectionPct;
+        minProtectedAssetsPct = _minProtectedAssetsPct;
 
         underlying.approve(_stabilityPool, type(uint256).max);
     }
 
     /**
-     * Set the minimum principal protection percentage.
+     * Set the minimum protected assets percentage.
      *
-     * @param _pct The new minimum principal protection percentage.
+     * @param _pct The new minimum assets protection percentage.
      */
-    function setMinPrincipalProtectionPct(uint16 _pct) external onlySettings {
-        minPrincipalProtectionPct = _pct;
+    function setMinProtectedAssetsPct(uint16 _pct) external onlySettings {
```

Comment:
Should the minProtectedAssetsPct have a minimum and a maximum?

Reply:
Hmm, not really sure tbh